model/relabel/relabel.go

```diff
@@ -255,6 +255,17 @@ func relabel(cfg *Config, lb *labels.Builder) (keep bool) {
 			return false
 		}
 	case Replace:
+		// Fast path to add label pair.
```

Comment:
Maybe this should say "add or delete", since one path leads to `Del()` ?

Reply:
Yes.. this logic is a copy from the below, the invalid label is dropped.